### PR TITLE
feature/AT-UN-18-jello-submarine

### PR DESCRIPTION
**Pull request by Yairmar**

### CODE REVIEW

**Review by fasaga — APPROVED** [review]

File: src/routes/at-university.routes.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 const router = require("express").Router();
-const { getAllCourses } = require ('../controllers/admin.controller'); 
+
 const { renderCourses } = require ('../controllers/at-university.controller');
 
 /* Routes */
 
 //Get Dashboard
```
Comment (at-internship) [inline src/routes/at-university.routes.js]: Should be // AT-UNIVERSITY-UI - Dashboard

File: src/routes/admin.routes.js [hunk 2]
```diff
@@ -10,5 +10,42 @@ router.get('/course', renderCourseForm)
...
+    addCourse,
+    renderEditCourseForm,
+    updateCourse,
+    deleteCourse
+} = require("../controllers/admin.controller");
+
+// ============= Sub Routes =============
+
+// AT-UNIVERSITY-UI - Admin - Index
+router.get("/admin", renderIndexAdmin);
```
Comment (at-internship) [inline src/routes/admin.routes.js]: Should be `router.get("/", renderIndexAdmin);`

File: src/routes/admin.routes.js [hunk 3]
```diff
@@ -10,5 +10,42 @@ router.get('/course', renderCourseForm)
...
+
+
+// AT-UNIVERSITY-UI - Admin - Users - Render Course List
+router.get("/admin/course", renderCourseList);
```
Comment (at-internship) [inline src/routes/admin.routes.js]: Should be `router.get("/course", renderCourseList);`

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**